### PR TITLE
Feature/toast messages

### PR DESCRIPTION
**Pull request by IsabellUrban**

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -5,16 +5,30 @@ import collections from "@/assets/collections.json";
...
   });
 
+  const [toastMessage, setToastMessage] = useState(null);
   const [currentFlashcard, setCurrentFlashcard] = useState(null);
 
   const [actionMode, setActionMode] = useState("default");
 
+  function showToastMessage(message, variant, icon) {
+    setToastMessage({ message, variant, icon });
+    setTimeout(() => {
```
Comment (Inelukas) [inline pages/_app.js:23]: Consider using the useEffect hook to manage the timeout, ensuring it clears properly.

**Review by Inelukas — COMMENTED** [review]
Hi! Overall, everything's working fine and the popup looks great! But there's a small bug that happens when clicking a popup away before the time runs out. After doing so, every following pop-up will disappear at he wrong time, usually before 5.5 seconds have passed.

**Comment by IsabellUrban** [comment]
> Hi! Overall, everything's working fine and the popup looks great! But there's a small bug that happens when clicking a popup away before the time runs out. After doing so, every following pop-up will disappear at he wrong time, usually before 5.5 seconds have passed.

Hi, thanks I implemented the useEffect. This should also fix the bug.

**Comment by Inelukas** [comment]
Great work! Your code looks clean and well-organized and everything's working as expected 😊
Actually I wasn't expecting you to post this story again in the CR as I already gave you a green tick in the QA thread ^^

**Pull request merged**